tests/debug.rs

```diff
@@ -0,0 +1,8 @@
+#[path = "debug/dump.rs"]
```

Comment:
I think it may be best to rename this file to `tests/debug/main.rs`, and then I think you can remove the `#[path]` annotations